src/Cli/dotnet/Commands/Test/TestingPlatformCommand.cs

```diff
@@ -106,7 +119,7 @@ private void SetupCancelKeyPressHandler()
         Console.CancelKeyPress += (s, e) =>
         {
             _output?.StartCancelling();
-            CompleteRun();
+            CompleteRun(exitCode: 3);
```

Comment:
We have an enum where the exit code should be added I think.

Reply:
Added to the enum